squad/frontend/templates/squad/test_history.jinja2

```diff
@@ -43,7 +43,7 @@
               <td class='{{result.status|slugify}}'>
                 <a href="{{project_url(result.test_run)}}">{{result.status}}</a>
                 {% if result.info['test_description'] or result.info['suite_instructions'] or result.info['test_instructions'] or result.info['test_log'] %}
-                  <a href='#' data-toggle="modal" data-target="#info_modal" data-info="{{ result.info }}"><span data-toggle="tooltip" data-placement="right" title="{{ _('Show info') }}" class='fa fa-info-circle'></span></a>
+                  <a href='#' data-toggle="modal" data-target="#info_modal" data-info="{{ to_json(result.info) }}"><span data-toggle="tooltip" data-placement="right" title="{{ _('Show info') }}" class='fa fa-info-circle'></span></a>
```

Comment:
it seems that json.dumps will always use double quotes for strings, so AFAICT for this to work the HTML should be changed to use single quotes?

Reply:
That's true, but Jinja will render those into html entities. The javascript bit of this page will convert them back to quotes when interpreting such string back to JS objects.